packages/ui/src/views/w3m-account-view/index.ts

```diff
@@ -0,0 +1,116 @@
+import { AccountCtrl, ClientCtrl, ConnectModalCtrl } from '@web3modal/core'
+import { html, LitElement } from 'lit'
+import { customElement, state } from 'lit/decorators.js'
+import '../../components/w3m-button'
+import '../../components/w3m-modal-footer'
+import '../../components/w3m-text'
+import '../../components/w3m-zorb-ens-image'
+import { CLIPBOARD_BLUE, CONNECTED_INDICATOR, DISCONNECT, ETH_LOGO } from '../../utils/Svgs'
+import { global } from '../../utils/Theme'
+import { formatAddress, roundBalance } from '../../utils/UiHelpers'
+import styles, { dynamicStyles } from './styles'
+
+@customElement('w3m-account-view')
+export class W3mAccountView extends LitElement {
+  public static styles = [global, styles]
+
+  // -- state & properties ------------------------------------------- //
+  @state() private address = ''
+  @state() private balance = ''
+  @state() private ensAvatar = ''
+
+  // -- lifecycle ---------------------------------------------------- //
+  public constructor() {
+    super()
+    this.subscribeAccountChanges()
+  }
+
+  // -- private ------------------------------------------------------ //
+  private unsubscribe?: () => void = undefined
+
+  private subscribeAccountChanges() {
+    this.address = AccountCtrl.state.address
+    this.balance = AccountCtrl.state.balance
+    this.ensAvatar = AccountCtrl.state.ensAvatar
+
+    this.unsubscribe = AccountCtrl.subscribe(() => {
+      this.address = AccountCtrl.state.address
+      this.ensAvatar = AccountCtrl.state.ensAvatar
+      this.balance = AccountCtrl.state.balance
+    })
+
+    return () => this.unsubscribe
```

Comment:
Same as above, this return is not needed and proper unsubscribing needs to be done in appropriate lit lifecycle method.